repository_service_tuf/helpers/api_client.py

```diff
@@ -88,24 +89,38 @@ def is_logged(settings: LazySettings):
         )
 
 
-def get_headers(settings: LazySettings) -> Dict[str, str]:
-    if settings.get("AUTH") is False:
+def get_headers(
+    settings: LazySettings, token: Optional[str] = None
+) -> Dict[str, str]:
+
+    built_in_auth: bool = settings.get("AUTH")
+
+    if built_in_auth is False and token is None:
         return {}
 
     server = settings.get("SERVER")
-    token = settings.get("TOKEN")
+    token = settings.get("TOKEN") or token
```

Comment:
Verifying the opposite, the `token` and the `settings` is better.get("TOKEN").
There is a situation where the user has `.rstuf.ini` with the TOKEN, but he didn't call `--auth`; in that scenario, it will load the token from the setting file.
```suggestion
    token = token or settings.get("TOKEN")
```

Reply:
Good catch! Hm, but we are still missing a case, from #39 we have that: 
> ... if the user uses `rstuf --auth` it ignores the token parameter.

this means that the priority is --`auth` > `--token`, if both are there, which makes sense. But with the suggested snippet `token` will be chosen. I have changed the code to take care of this case as well.